iree/turbine/aot/compiled_module.py

```diff
@@ -107,12 +112,27 @@ def __call__(self, *args, **kwargs):
         return self.py_value(*args, **kwargs)
 
 
+class ExportTargetDef:
```

Comment:
I am not sure if I should drop the `ExportTargetDef` and use half-initialized ExportProcDef and ExportedProgramDef directly.

Reply:
What is the benefit to having this separate class?

Reply:
In the place where we need a structure to store this data we are not ready to construct ExportProcDef or ExportedProgramDef. If we are to use them directly they will have a more complicated multi-step initialization.